### ISSUE TITLE
Can't get NVDA to read selected in tree browser

### ISSUE THREAD

**Issue by Ardena** [nvaccess/nvda#8587]
### Steps to reproduce:
1. Open Firefox
2. Go to https://codepen.io/Ardena/pen/oMdMgq
3. Navigate to the tree browser in the rendered area
4. The item "Advanced Math Assignments" is marked as aria-selected in the DOM.  When you enter forms mode for the tree, it should automatically take you to the selected element.  (It has tabindex="0" as is described at https://www.w3.org/TR/wai-aria-practices-1.1/#TreeView. while everything else in the tree has tabindex="-1") Notice when you enter the tree that it leaves you at the top level
5. Notice that no matter what you do, the aria-selected item doesn't announce that it's selected.

### Actual behavior:
When you have a tree item that has aria-selected="true", you are never told that anything is selected in the tree and when you enter the tree, you aren't directed to the selected item even if it's the item with the correct tab index.

### Expected behavior:
The tree forms mode should take you to the selected item and there should be some indication that the item is selected.  Also according to the w3 link above, in a single select tree (which is what I'm trying to build), it says you should not use aria-selected="false" on items that aren't selected, so it should read selected if the item is selected.

I have tried setting up the tree with aria-selected="false" for the items that aren't selected, and that does read out "not selected" for those items, so I'll use that workaround for now.

#### NVDA Installed/portable/running from source:
NVDA is installed

#### NVDA version:
Version 2018.2.1

#### Windows version:
Windows 10 Pro Version 10.0.10586 Build 10586

#### Name and version of other software in use when reproducing the issue:
Firefox 61.0.1

#### Other information about your system:
Windows virtual machine using VirtualBox

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
This is actually reproducible in every browser with NVDA 2019.1.1. See also referenced issue above.
cc: @jcsteh, @feerrenrut

**Comment by seanbudd** [comment]
Closing as stale, #14352 is a newer duplicate

**Comment by Adriani90** [comment]
Reopening, since the issue that is described here is about focus management even though the title is not clear enough.
Two issues are described here:
1. The first issue about aria-selected=true: NVDA doesn't report "selected" but it reports "not selected" on items with aria-selected=false. This is consistent with e.g. Windows file explorer and is intended behavior.
2. The issue that the system focus is not redirected to the correct tree item "e.g. advanced math assignments", when focusing it with the virtual browse mode cursor and pressing NVDA+space bar to enable focus mode. The system focus is not synchronized properly and is not redirected to the advanced math assignments, but to the top of the tree view. cc: @jcsteh any suggestions on the focus management part of this issue? Is this web authoring error?
